daphne/src/roles.rs

```diff
@@ -313,7 +348,7 @@ pub trait DapLeader<S>: DapAuthorizedSender<S> + DapAggregator<S> {
             self,
             task_id,
             task_config,
-            "/aggregate",
+            format!("/{}/{}", task_config.version, "aggregate").as_str(),
```

Comment:
This breaks interop, since not every helper is going to implement versioning the way we do. (Here and below.) Instead of changing how the endpoints are constructed here, the helper URL in the task configuration will need to be changed.

Reply:
I've removed all the endpoint construction using versions.